### DISCUSSION TITLE
Incompatible mods found!

### DISCUSSION THREAD

**Discussion by SERLEN80** [FabricMC/fabric-api#3705] [Mod Dev Support] [answered]
### Description

Incompatible mods found! net.fabricmc.loader.impl.FormattedException: Some of your mods are incompatible with the game or each other! A potential solution has been determined, this may resolve your problem: 	 - Replace 'Minecraft' (minecraft) 1.20 with version 1.20.3 or version 1.20.4. More details: 	 - Mod 'Iris' (iris) 1.6.14 requires version 1.20.3 or version 1.20.4 of 'Minecraft' (minecraft), but only the wrong version is present: 1.20! 	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:51) 	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:195) 	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:146) 	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68) 	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

**Comment by IMS212** [comment, marked as answer]
Iris is incompatible with 1.20.0.

**Reply by weranki** [reply]
ok

**Reply by hansenliam-dot** [reply]
create fabric has a problom

**Comment by Osvaldo970** [comment]
Mod 'HerosAnchorOptimizer' (herosanchoroptimizer) 1.1.2 requires any version of fabric, which is missing!
Incompatible mods found!
net.fabricmc.loader.impl.FormattedException: Some of your mods are incompatible with the game or each other!
A potential solution has been determined, this may resolve your problem:
	 - Install fabric, any version.
More details:
	 - Mod 'HerosAnchorOptimizer' (herosanchoroptimizer) 1.1.2 requires any version of fabric, which is missing!
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:51)
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:202)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:142)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:66)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
What should I do?

**Reply by Poopooracoocoo** [reply]
Please make your own post next time. 

You need to get a mod called Fabric API.

**Comment by edwinfalk7-lab** [comment]
Bro i cant download this mod https for 1.21.11 idk what mod i need for it                                                                                                                               https://modrinth.com/mod/armor-durability-hud?version=1.21.11&loader=fabric

**Comment by Radin337** [comment]
Incompatible mods found!
net.fabricmc.loader.impl.FormattedException: Some of your mods are incompatible with the game or each other!
A potential solution has been determined, this may resolve your problem:
	 - Replace mod 'Meteor Client' (meteor-client) 26.1.2-42 with any version that is compatible with:
		 - minecraft 26.2
More details:
	 - Mod 'Meteor Client' (meteor-client) 26.1.2-42 requires any version between 26.1 (inclusive) and 26.2- (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 26.2!
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:51)
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:202)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:142)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:66)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
